Importer.pm

```diff
@@ -383,9 +375,22 @@ sub _prepareTrack {
 		push @$artistIds, 'qobuz:artist:' . $track->{performer}->{id};
 	}
 
-	my $rolePerformer;
+	$attributes->{ARTIST} = \@$artists;
+	$attributes->{ARTIST_EXTID} = \@$artistIds;
```

Comment:
Assuming `$artists` was a list reference, then prefixing it with `@$artists` would de-reference it, and `\@$artists` would reference the de-referenced list - resulting in the same again? Or is this a trick to clone the list?